components/entitlement/org.wso2.carbon.identity.entitlement.endpoint/src/main/java/org/wso2/carbon/identity/entitlement/endpoint/resources/AbstractResource.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.entitlement.endpoint.resources;
+
+import org.wso2.carbon.identity.entitlement.endpoint.util.EntitlementEndpointConstants;
+
+/**
+ * Abstract class for service resource
+ */
+public class AbstractResource {
+    public String identifyOutputFormat(String format) {
+        if (format == null || ("*/*").equals(format) || ((format != null) && (format.startsWith(EntitlementEndpointConstants.APPLICATION_JSON)))) {
```

Comment:
* Maintain 120 character limit for each line.
* Condition can be simplified to, 
`format == null || ("*/*").equals(format) || format.startsWith(EntitlementEndpointConstants.APPLICATION_JSON)`